EasyPost.Tests/ErrorTest.cs

```diff
@@ -29,10 +36,107 @@ public async Task TestBadParameters()
         }
 
         [Fact(Skip = "Test is no longer valid")]
-        public Task TestEmptyApiKey()
+        public void TestEmptyApiKey()
         {
+            UseVCR("empty_api_key");
+
             // No longer possible to have an empty API key
+        }
+
+        [Fact]
+        public Task TestKnownApiExceptionGeneration()
+        {
+            UseVCR("known_api_exception_generation");
```

Comment:
Question: What's with the `UseVCR` function here? This doesn't even use VCR... Kinda misleading.

Reply:
Fair enough, just wanted to remain consistent in case this unit test did make API calls in the future.